homeassistant/components/repetier/sensor.py

```diff
@@ -37,11 +47,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         temp_id = info["temp_id"]
         name = info["name"]
         if temp_id is not None:
-            name = "{}{}{}".format(name, SENSOR_TYPES[sensor_type][3], temp_id)
+            _LOGGER.debug("%s Temp_id: %s", sensor_type, ent_id[sensor_type])
+            name = "{}{}{}".format(
```

Comment:
Please try to use f-strings here:
```python
name = f"{name}{SENSOR_TYPES[sensor_type][3]}{ent_id[sensor_type]}"
```
https://www.python.org/dev/peps/pep-0498/

Reply:
Done